ghcide/ghcide.cabal

```diff
@@ -83,8 +83,10 @@ library
     , hiedb                        ^>= 0.6.0.0
     , hls-graph                    == 2.9.0.1
     , hls-plugin-api               == 2.9.0.1
+    , http-conduit
```

Comment:
Let's split out the `http` stuff, it is unrelated (but a welcome addition in general) to the structured diagnostics